test/tracing/tracer_test.rb

```diff
@@ -51,9 +51,7 @@ def test_exotic_tag_types
 
     assert_equal false, ::Instana.tracer.tracing?
 
-    require 'resolv'
-    r = Resolv::DNS.new
-    ipv4 = r.getaddress("www.pwpush.com")
+    ipv4 = '185.15.59.224'
```

Comment:
Just a thought if the value of ipv4 does not matter, then can we use data which actually suggests that its for testing purposes only .. Like 111.111.111.111 or some value which suggests that it is just for testing and does not have any significance. Just for better readability

Reply:
Yeah, why not.